README.md

```diff
@@ -153,3 +153,30 @@ with:
         post: true
         debug: false
 ```
+
+## How to encrypt the `evmPrivateEncrypted` parameter
+
+Partner private key (`evmPrivateEncrypted` config param in `conversation-rewards` plugin) supports 3 formats:
+1. `PRIVATE_KEY` (allowed to be used only in `ubiquity` and `ubiquibot` organizations)
+2. `PRIVATE_KEY:GITHUB_ORGANIZATION_ID`
+3. `PRIVATE_KEY:GITHUB_ORGANIZATION_ID:GITHUB_REPOSITORY_ID`
+
+Format `PRIVATE_KEY` can be used only for `ubiquity` and `ubiquibot` organizations. It is kept for backwards compatibility in order not to update private key formats for our existing values set in the `evmPrivateEncrypted` param.
+   
+Format `PRIVATE_KEY:GITHUB_ORGANIZATION_ID` restricts in which particular organization this private key can be used. It can be set either in the organization wide config either in the repository wide one.
```

Comment:
The intermediary `PRIVATE_KEY:GITHUB_ORGANIZATION_ID` format is meant to be set in the org wide ubiquibot config.

If we support only the max secure `PRIVATE_KEY:GITHUB_ORGANIZATION_ID:GITHUB_REPOSITORY_ID` format then **for all of our repositories** we'll have to:
1. Create a separate config for the `conversation-rewards` plugin
2. Create a separate `evmPrivateEncrypted` param because repository IDs are different